routes/x/index.tsx

```diff
@@ -279,6 +279,20 @@ export default function ThirdPartyRegistryList(
               </p>
             </div>
 
+            <div class={tw`space-y-3`}>
+              <h2 class={tw`text-xl leading-6 font-semibold`}>
+                A module is name-squatting or its just made as a joke, can I
+                have it?
+              </h2>
+              <p class={tw`text-[#6C6E78] leading-5`}>
+                if there is a module that has been deemed as "low quality" or
```

Comment:
Name squatting is not allowed in the deno.land/x registry. 
If you feel that a module is not currently usable, has has not been legitimately under development for more than 90 days, and you have a concrete proposal to publish a well-maintained module in its place, please contact support.